src/context/WalletProvider/KeepKey/components/RecoverySentenceEntry.tsx

```diff
@@ -43,6 +43,7 @@ export const KeepKeyRecoverySentenceEntry = () => {
   const [characterInputValues, setCharacterInputValues] = useState(
     Object.seal(new Array<string | undefined>(maxInputLength).fill(undefined)),
   )
+  const [passphrase, setPassphrase] = useState<Array<Array<string | undefined>>>()
```

Comment:
Note: We usually use the `type[]` syntax around the codebase, but I won't block the PR for this - feel free to tackle as a follow-up commit at your discretion and I will re-stamp @brymut.
```suggestion
  const [passphrase, setPassphrase] = useState<(string | undefined)[][]>()
```
If you decide to commit those changes and push them without clicking the suggested changes button, make sure not to force push so it can be easily re-approved.